pkg/processor/build/builder.go

```diff
@@ -591,6 +606,15 @@ func (b *Builder) getObjectsToCopyToProcessorImage() map[string]string {
 		objectsToCopyToProcessorImage[path.Base(localObjectPath)] = dockerObjectPath
 	}
 
+	filesToCopy := b.processorImage.filesToCopy
+	if b.processorImage.scriptPathToRunDuringBuild != "" {
+		filesToCopy = append(filesToCopy, b.processorImage.scriptPathToRunDuringBuild)
+	}
+
+	for _, fileToCopy := range filesToCopy {
+		objectsToCopyToProcessorImage[path.Base(fileToCopy)] = "/opt/nuclio"
```

Comment:
`objectPathsToCopyToProcessorImage` (i know this is something I incorrectly named :)

Reply:
OK